examples/03-Maxwell/Maxwell2D_PMSynchronousMotor.py

```diff
@@ -496,35 +493,33 @@ def create_cs_magnets(pm_id, cs_name, point_direction):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Define windings in phase A.
 
-M2D.assign_coil(input_object=["Coil"], conductor_number=6, polarity="Positive", name="CT_Ph1_P2_C1_Go")
-M2D.assign_coil(input_object=["Coil_5"], conductor_number=6, polarity="Negative", name="CT_Ph1_P2_C1_Ret")
-M2D.assign_winding(coil_terminals=None, winding_type="Current", is_solid=False,
-                   current_value=PhA_current, parallel_branches=1, name="Phase_A")
-M2D.add_winding_coils(windingname="Phase_A", coil_names=["CT_Ph1_P2_C1_Go", "CT_Ph1_P2_C1_Ret"])
+M2D.assign_coil(objects=["Coil"], conductors=6, polarity="Positive", name="CT_Ph1_P2_C1_Go")
```

Comment:
I would use conductors_number instead of conductors. Having conductors as argument I expect the argument is a list